scratch/PCA/extract_data.ipynb

```diff
@@ -18,40 +18,81 @@
     "\"\"\"\n",
     "\n",
     "from __future__ import division\n",
```

Comment:
Branden, can you meet briefly?
--
Henry




> On Aug 16, 2016, at 11:05 AM, bkappes <notifications@github.com> wrote:
> 
> In scratch/PCA/extract_data.ipynb <https://github.com/csm-adapt/tracr/pull/4#discussion_r74976937>:
> 
> > @@ -18,40 +18,81 @@
> >      "\"\"\"\n",
> >      "\n",
> >      "from __future__ import division\n",
> Yes. I’m in my office.
> … <x-msg://5/#>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
>  <https://github.com/csm-adapt/tracr/pull/4#discussion_r74976256> <https://github.com/csm-adapt/tracr/pull/4/files/c34dbfd1457b395e51fb01c9960400ba80f95fbc#r74976256> <https://github.com/notifications/unsubscribe-auth/AFjEmRkrgnsPPTOpTYPOy43qCZgect5pks5qgez7gaJpZM4Jk3y6>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/csm-adapt/tracr/pull/4/files/c34dbfd1457b395e51fb01c9960400ba80f95fbc#r74976937>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AMZvXbjt61Nz-r-5qKqcwrHmzvwTlp4cks5qge3ogaJpZM4Jk3y6>.
>